src/pngwriter.h

```diff
@@ -732,6 +738,12 @@ class pngwriter
 
    int static get_text_width_utf8(char * face_path, int fontsize, char * text);
 
+   int fillBackgroundColor(void);
```

Comment:
Can you please add (doxygen) header comments for all of those?

Should they really be public (aka, why do we need to extent the API)?